.github/workflows/ci.yml

```diff
@@ -77,11 +77,11 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          key: shardingsphere-it-cache-${{ github.sha }}
```

Comment:
Maybe `hashFiles('**/pom.xml')` is better than `github.sha`, since `github.sha` is changed frequently, but `hashFiles('**/pom.xml')` might be not changed, so cache could be reused.

Reply:
you are right, will refactor in next PR.